tools/toitlsp/lsp/toitdoc/inheritance/inheritance.go

```diff
@@ -347,27 +347,6 @@ func (ir *InheritanceResult) doClass(cls *toit.Class) {
 		return
 	}
 
-	superClass := ir.resolveClassRef(cls.SuperClass)
```

Comment:
Slight reshuffle, but fundamentally, we do the same thing we did for the superclass now also for the mixin classes (in a loop).